File: 02-C/04-condicionales/01ifElse.c
// IF Else en C
#include <stdio.h>

int main()
{
    int numero;
    printf("Ingresa un número: \n");
    scanf("%d", &numero);

    // If
    // Verificar si el numero es positivo.
    if (numero > 0)
    {
        printf("Valor positivo = %d", numero);
    }
    // Else if
    else if (numero < 0)
    {
        printf("Valor negativo = %d", numero);
    }
    else
    {
        printf("Valor 0 = %d", numero);
    }

    // Operador ternario: Nos permite simplificar un if else solamente. Se recomienda usar cuando solo sea una linea de código. 
    //Verdadero : Falso
    (numero > 0) ? printf("Valor positivo") : printf("Valor 0 o negativo");


return 0;
}

File: 04_CSHARP/04_Condicionales/01ifElse.cs
// If Else en C#

int numero;
Console.WriteLine("Ingresa un numero");
numero = Convert.ToInt32(Console.ReadLine());

// Verificar si el valor es positivo 
// If else - Else if 
if (numero > 0)
{
    Console.WriteLine($"Valor positivo: {numero}");
}
else if (numero < 0)
{
    Console.WriteLine($"Valor negativo: {numero}");
}
else
{
    Console.WriteLine($"Valor 0: {numero}");
}


// Operador ternario: Permite simplificar un if else solamente. Se recomienda usar cuando solo sea una linea de código.

// Verdadero : Falso
var resultado = (numero > 0) ? "Positivo" : "Negativo o 0";
Console.WriteLine($"Valor: {resultado}");
